src/ASM/ASMs2D.C

```diff
@@ -1082,8 +1082,32 @@ void ASMs2D::constrainCorner (int I, int J, int dof, int code, char basis)
   if (basis < 1) basis = 1;
 
   int node = this->getCorner(I,J,basis);
-  if (node > 0)
```

Comment:
(ignore line number); s/Tweek/Tweak/g in commit msg.

Reply:
ok, corrected.